.ado/templates/e2e-test-job.yml

```diff
@@ -28,6 +28,29 @@ jobs:
             
       - template: prepare-env.yml
 
+      - task: CmdLine@2
```

Comment:
Could we try 0.64 without all these nuget commands? Maybe just the nuget.config files?
That way we know that we can clean them in in the main branch too..

Reply:
I tried that with the initial commit thinking the same thing, seems like adding the nuget.congifs alone was not enough. I am curious though if `nuget sources list` before the restore is enough on it's own, I'll try removing the cache and solution cleans.